jwst/emicorr/emicorr.py

```diff
@@ -0,0 +1,792 @@
+#
+#  Module for applying MIRI EMI correction
+#
+
+import numpy as np
+import logging
+
+from stdatamodels.jwst import datamodels
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+default_emi_freqs = {
+    "Hz390": 390.625,
```

Comment:
Curious as to why it's necessary to have multiple entries for 390 and 10, when they all have the same value?

Reply:
because they have a different phase amplitude according to the subarray-detector-readpatt configuration.